File: benchmark/PHP 7.0/字符串连接性能测试.php
<?php

$t = microtime(true);

$s = "";
$a = "foo";
$b = "bar";
$c = "ok";
for ($i = 0; $i < 1000000; ++$i)
	$s = $a.$b.$c.$i;

echo (microtime(true) - $t), "\n";
echo $s, "\n";

File: benchmark/Lua 5.3/字符串连接性能测试.lua
local start = os.clock()

local s
local a = "foo"
local b = "bar"
local c = "ok"
for i = 0, 1000000-1, 1 do
	s = a..b..c..i
end

print(os.clock()-start, "\n")
print(s, "\n");
